src/context.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ContextWatchdog from '@ckeditor/ckeditor5-watchdog/src/contextwatchdog';
+
+export const ContextWatchdogContext = React.createContext( 'contextWatchdog' );
```

Comment:
Is it safe if a user will use two `<Context>` components on the same page?

Reply:
If so, WDYT about adding a getter to the Context component that will return the value?

Reply:
This context uses later a provider. So in theory you could use such structure:

```
<Context>
  <Editor />
</Context>
<Context>
  <Editor />
</Context>
```

And everything should work.

Reply:
👍